pages/styles/about.css

```diff
@@ -0,0 +1,88 @@
+
+* {
+    font-family: sans-serif;
+}
+
+header {
+    justify-content: left;
+    display: flex;
+    height: 300px;
+    order: 2;
+    flex-direction: column;
+    flex-shrink: 0;
+    width: 200px; 
+    align-self: flex-start;
+    margin: 40px;
+}
+
+
+h3 {
+    justify-content: center;
+    display: flex;
+    order: 3;
+    flex-direction: column;
+    flex-shrink: 0;
+    width: 600px; 
+    flex-grow: 2;
+    flex-wrap: wrap;
+    height: 80px;
+    align-content: space-between;
+    margin: 50px 50px 50px 250px;  
+}
+
+nav {
+    bottom: 250px;
+    display: flex;
+    flex-direction: row-reverse;
+    font-size: 25px;
+    position: relative;
+    left: 740px;
+    width: 500px;
+    margin: auto; 
+    order: 1;
+}
```

Comment:
For this page, both of these are in the `header` element, which means that they will inherit their properties from it. As such, you don't need to repeat the `display`, `flex-direction`, and `flex-shrink` properties as they match the values in the `header` rule above.